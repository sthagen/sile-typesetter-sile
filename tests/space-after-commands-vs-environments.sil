\begin[papersize=a6]{document}
\nofolios
\neverindent
\noop{aaa}

bbb

\begin{noop}
aaa
\end{noop}

bbb
\end{document}
