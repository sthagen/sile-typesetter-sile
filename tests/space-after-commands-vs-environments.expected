Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	25.6639
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	68 68 68 w=13.7695	(aaa)
My 	37.6639
T	69 69 69 w=15.2344	(bbb)
My 	49.6639
T	68 68 68 w=13.7695	(aaa)
My 	61.6639
T	69 69 69 w=15.2344	(bbb)
End page
Finish
